bin/setup-popsyn

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+# script to setup a population run with one or multiple metallicities
+# 1. Reads the args input file
+# 2. Determine the number of metallicities
+# 3. Create a submit file for each metallicity
+# 4. Create script to merge job array runs into a single population file 
+# 5. Create a file to submit all the metallicities
+# 6. Including the dependencies
+# Author: Max Briel
+import os
+from posydon.popsyn.io import binarypop_kwargs_from_ini
+from posydon.utils.common_functions import convert_metallicity_to_string
+from posydon.config import PATH_TO_POSYDON, PATH_TO_POSYDON_DATA
+import argparse
+
+def create_run_script(ini_file):
+    '''creates a run script for a specific metallicity'''
+   
+    text=f'''from posydon.popsyn.binarypopulation import BinaryPopulation
+from posydon.popsyn.io import binarypop_kwargs_from_ini
+from posydon.utils.common_functions import convert_metallicity_to_string
+import argparse
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('metallicity', type=float)
+    args = parser.parse_args()
+    ini_kw = binarypop_kwargs_from_ini("{ini_file}")
+    ini_kw["metallicity"] = args.metallicity
+    str_met = convert_metallicity_to_string(args.metallicity)
+    ini_kw["temp_directory"] = str_met+"_Zsun_" + ini_kw["temp_directory"]
+    synpop = BinaryPopulation(**ini_kw)
+    synpop.evolve()\n'''
```

Comment:
As far as I got it from @kkovlakas's code style guide we should use triple quotes only for doc strings.
An alternative would be to concatenate strings and use line break characters for multi line strings.

Reply:
The code style is not clear in this, except for not using it for comments.
A similar style is also used in some of the other pipeline scripts.

I've moved the function to posydon.popsyn.io for a cleaner look here.

Reply:
@maxbriel: no, in the (post-processing) pipeline triple strings are only used for docstrings. There are a few cases, where they are used in the grid running scripts. But there it looks like two people modified the script and inserted two styles. Hence, instead of unifying the style to one, your penetrating to uses different styles across the code.
We should agree on one style and use it always.